packages/pixeloven-cli/src/commands/lint.ts

```diff
@@ -26,6 +26,7 @@ export default {
             }
             return status;
         };
+        console.log(parameters.array);
```

Comment:
is this to help future debugs or is it temporary?

Reply:
Awe I'll remove